src/screens/Asset/ReceiveModal.js

```diff
@@ -173,6 +189,16 @@ const combinedMapStateToProps = (state: RootReducerState): $Shape<StateProps> =>
 
 export default (connect(combinedMapStateToProps)(ReceiveModal): AbstractComponent<OwnProps>);
 
+const styles = {
+  ensName: {
+    fontSize: fontSizes.big,
+  },
+  address: {
+    color: getThemeColors().basic030,
```

Comment:
getThemeColors it seems that `getThemeColors` expects theme to be passed and you only can pass them from inside the component as you get it from hook/context. Solution here would be to apply `textStyle={[styles.address, color: colors.basic030]}`

Regarding color values, pls use `secondaryText` instead of `basic030` as it's the same hex value for both light & dark but it's easier to read.

Reply:
@divyaBacancy @mdjastrzebski Point 1 definitely makes sense - however we may need to be specific when we say `All chains` because there will always be someone who takes this to mean that they can use their crazy EVM blockchain that we don't support on Etherspot 😂 but yes this message should be hidden altogether from Archanova accounts. Maybe instead of `All chains use single wallet address` we could re-word it to something like `Our supported Ethereum blockchains (mainnet, Matic, ...) use the above address` - just a suggestion.

Point 2 works in both scenarios.

Reply:
@divyaBacancy @mdjastrzebski on further reflection - i'm going to consult the design and product team on how we should approach this message. It might require design rework. Let's skip this for now.

Reply:
@IAmKio @mdjastrzebski @poocart Done with the changes.